Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using LitJson;
+using System.IO;
+using UnityEngine.SceneManagement;
+
+public class GameManager : MonoBehaviour
+{
+    public static GameManager instance = null;
+
+    public string readJson;
+
+    void Awake()
+    {
+        if (instance == null)
+            instance = this;
+
+        else if (instance != this)
+            Destroy(gameObject);
```

Comment:
Please explain why you destroy gameObject here not instance?

Reply:
When we change scene, existing class's instance is this. But new class's instance is not this. So, the gameobject that has new class is destroy.